cPP_DSC.adoc

```diff
@@ -1080,7 +1122,7 @@ FCS_COP.1.1/SKC:: The TSF shall perform _symmetric-key encryption/decryption_ in
 {empty}[*selection*: [.underline]#ISO/IEC 10116:2017 (Clause 7), NIST SP 800-38A#] [CBC]
 
 |AES-CCM
-|AES in CCM mode with unpredictable, non-repeating nonce, minimum size of 64 bits
```

Comment:
I don't think unpredictable nonces are required for CCM. Section 5.3 of SP 800-38C says
>The nonce shall be non-repeating in the sense that any two distinct data pairs to be protected by CCM during the lifetime of the key shall be assigned distinct nonces. In effect, the nonce determines an invocation of CCM. The nonce is not required to be random.

As CCM is based on CTR (similar to GCM) I wouldn't expect an unpredictable nonce to be required either.

Reply:
Resolved in #308.